tests/end-to-end/regression/5172.phpt

```diff
@@ -20,7 +20,7 @@ Configuration: %s
 Time: %s, Memory: %s
 
 Issue5172 (PHPUnit\TestFixture\Issue5172)
- ✔ One
+ ? One
```

Comment:
What does the `?` mean here?

Reply:
Not sure, dont remember, but we are checking that; a test with a deprecated config 9.5 schema file reports the deprecation.

I believe it's coming from https://github.com/sebastianbergmann/phpunit/blob/ccb711a9b71d02741712961bdd1d7e519df9e460/src/TextUI/Output/TestDox/ResultPrinter.php#L378